tests/unit/test_experiments/test_experiment_steps.py

```diff
@@ -307,3 +307,28 @@ def custom_step_voltage(variables):
 
         with pytest.raises(ValueError, match="control must be"):
             pybamm.step.CustomStepImplicit(custom_step_voltage, control="bla")
+
+    def test_steps_with_operators(self):
+        # voltage
+        step = pybamm.step.voltage(1, duration=3600)
+        termination_lt_4_1 = pybamm.step.VoltageTermination(4.1, operator="<")
+        termination_gt_4_1 = pybamm.step.VoltageTermination(4.1, operator=">")
```

Comment:
we could use operator overloading and have the interface be `pybamm.step.termination.Voltage > 4.1` (overload the `__gt__` operator to return `pybamm.step.VoltageTermination(4.1, operator="<")`